README.md

```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
 
 ## Review Process for Original Posts
-- The review team will provide comments and edits. 
-- Once the review team has made suggested edits, they will share all suggested edits with the author who has final sign off on content.
-- If the author feels comfortable, we would also like to list out the names of the technical editors of the post.
-- The review team will share all suggested edits with the author, who has final sign off on content.  
-- Zibby Keaton will then post to the Node.js Foundation Medium publication.
-- The review process may take between two to three weeks depending on the number of edits and revisions involved. 
-
-## Review Process for Created Medium Posts
-- Members from the evangelism working group are invited to become editors on Medium. If interested, contact Zibby Keaton here: zkeaton@linuxfoundation.org.
-- Editors are allowed to request to put already available public content on the Node.js Foundation Medium publication if they follow the above guidelines. 
-- If you are interested in becoming a technical editor, please file an issue.
+- Writers submit an abstract via email at medium@nodejs.org.
+
+- Editors have one week to provide approval or feedback - this is done internally on the alias and not to be shared with the writer until a final decision and suggested changes are reached as a group.
+
+- If the abstract is approved, the writer will be informed and a task will be created on the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) for the post
+
+- The contributor will be made a “Writer” on the Node.js Collection medium account.
+
+- The contributor will write the first draft and Write will submit this draft on Medium and an editor or two will be assigned to review. The contributor will ping the medium alias to alert the editors a draft has been submitted. Clear directions on how to submit will be provided to the contributor.
+
+- Editor from the Node.js Collection will review the post and provide the writer comments or feedback. This process can take up to a month to complete.
+
+- The writer to approve all content and send a note to medium@nodejs.org once finished.
+
+- Once the piece is complete, and the writer approves the content, an editor from Node.js PR team to review one final time and accept into publication.
+
+- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/ promo.
+
+- The writer will be informed about the approval and publication date.
+
+## Review Process for Existing Created Medium Posts (to be republished with permission on the Node.js Collection)
+
+- Members from the evangelism working group are invited to become editors on Medium. If interested, contact here: medium@nodejs.org
```

Comment:
There is no such thing as the Evangelism WG. This line should likely be deleted, as it doesn't seem to be relevant to the heading.

Reply:
I will update it as "Anyone is invited to ......"
Later, we can limit it to CommComm and TSC if needed.

Ok?

Reply:
SGTM